lib/parity/backup.rb

```diff
@@ -53,10 +53,15 @@ def download_remote_backup
     def restore_from_local_temp_backup
       Kernel.system(
         "pg_restore tmp/latest.backup --verbose --clean --no-acl --no-owner "\
-          "-d #{development_db} #{additional_args}",
+          "--dbname #{development_db} --jobs `#{number_of_cores}` "\
+          "#{additional_args}",
       )
     end
 
+    def number_of_cores
+      "getconf _NPROCESSORS_ONLN"
+    end
```

Comment:
I like the `NUMBER_OF_CORES` environment variable idea but am thinking through the developer steps needed, wondering if it would add a burden.

In that scenario, we'd add something to the README of Parity about exporting that variable in their `zshrc` / `bashrc`, etc.?

Thoughts on http://stackoverflow.com/a/23378780? `sysctl` for OS X, `lscpu` (physical, not logical) for Linux? Would that leave various BSDs out in the cold?